roofit/multiprocess/res/RooFit/MultiProcess/Job.h

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Why the header files are in the subdirectory multiprocess/res and not in multiprocess/inc ?

Reply:
That was requested by @Axel-Naumann in https://github.com/root-project/root/pull/9078#discussion_r738341165